README.md

```diff
@@ -1,2 +1,26 @@
-# skel
-Skeletal structure of an Autopilot Pattern application
+# Skeleton structure for an Autopilot Pattern blueprint
+
+1. Update PRJ_NAME in `Makefile` to be a meaningful name, lile `redis`...
+2. Update `Dockerfile` to source from the correct base image, install the
+   correct dependencies, and launch ContainerPilot with the proper primary
```

Comment:
I think this might be a diff that got committed? You've got `+` chars in here and an 80 col wrap.